src/EventStore.Client.PersistentSubscriptions/PersistentSubscriptionInfo.cs

```diff
@@ -0,0 +1,182 @@
+﻿using System;
+using System.Collections.Generic;
+using EventStore.Client.PersistentSubscriptions;
+using Google.Protobuf.Collections;
+
+namespace EventStore.Client {
+	/// <summary>
+	/// Provides the details for a persistent subscription
+	/// </summary>
+	/// <param name="EventSource">Name of the stream used for the persistent subscription</param>
+	/// <param name="GroupName">Name of the persistent subscription</param>
+	/// <param name="Status">Current state</param>
+	/// <param name="AveragePerSecond">The average amount of messages being processed per second</param>
+	/// <param name="TotalItems">The total amount of messages processed</param>
+	/// <param name="CountSinceLastMeasurement">The amount of messages processed since last measurement</param>
+	/// <param name="LastCheckpointedEventPosition">Position of the last checkpointed event</param>
+	/// <param name="LastKnownEventPosition">Position of the last known event</param>
+	/// <param name="StartFrom">The position to start subscribing from</param>
+	/// <param name="ReadBufferCount">Number of messages currently in the read buffer</param>
+	/// <param name="LiveBufferCount">Number of messages currently in the live buffer</param>
+	/// <param name="RetryBufferCount">Number of messages currently in the retry buffer</param>
+	/// <param name="TotalInFlightMessages">Total number of in-flight messages</param>
+	/// <param name="OutstandingMessagesCount">Number of outstanding messages</param>
+	/// <param name="ParkedMessageCount">Number of parked messages</param>
+	/// <param name="Connections">Details of current connections</param>
+	/// <param name="Settings">The settings used to create the persistant subscription</param>
+	public record PersistentSubscriptionInfo(string EventSource, string GroupName, PersistentSubscriptionState Status,
```

Comment:
**QUESTION:** I'm not sure about the convention, but it seems that those values like `AveragePerSecond`, `TotalItems` etc., cannot be negative. Maybe it'd be worth setting them to `ulong` and `uint` accordingly?